### PR TITLE
Remove Inapplicable Correspondence Statistics 

### LINKED ISSUE #566: Remove Inapplicable Correspondence Statistics
### What is the problem?

As Lichess does not support a Correspondence leaderboard, statistics such as "better than x% of players" should be removed.

### Steps to reproduce the bug

Go to a player's profile
Click on their correspondence stats, if they have any.

### What did you expect to happen?

Statistics not applicable to Lichess's implementation of correspondence chess should not appear.

### What happened instead?

The information present is not accurate, as Correspondence does not have a leaderboard system.

### Device Info

N/A

### Operating System Version

N/A

### Additional information

Screenshot
IMG_0966

### PR DESCRIPTION
**Pull request by BAW2501**
closes #566

currently the UserPerfStats has two problems

1. displays [USERNAME] is better than 0.00% of [`perfType`] players. when` perf.percentile == 0.0`
2. displays ? in Rank Card for perfs with no leaderboard (i.e. correspondance)

proposed fix

1. display AreBetterThanPercentOfPerfTypePlayers when `perf.percentile > 0.0`.
2. remove rank `StatCard` for correspondence perf.

**maintainer input needed**
when removing rank stat card visually going from 2×2 grid layout to 1 then 2×1  isn't pleasing is it okay if we leave it as is or should we remove rows all together (for correspondence )  to display progress, rating deviation ,highest Elo and lowest Elo on a single row
in short

the original is

| Progress    |            |
| ----------- | ---------- |
| rank        | deviation  |
| highest elo | lowest elo |

<table>
<tr><th>Layout 1 </th><th>Layout 2</th></tr>
<tr><td>

| Progress    |            |
| ----------- | ---------- |
| deviation   |            |
| highest elo | lowest elo |

</td><td>

| Progress    |
| ----------- |
| deviation   |
| highest elo |
| lowest elo  |

</td></tr> </table>

### CODE REVIEW

**Comment by veloce** [comment]
I'd keep the 2x2 grid. Why not showing deviation aside to progress?

**Comment by BAW2501** [comment]
> I'd keep the 2x2 grid. Why not showing deviation aside to progress?

yeah that works too , I'll change it accordingly
